tests/codes/cur/cur_228/test_formula_2_22.py

```diff
@@ -0,0 +1,42 @@
+"""Test for formula 2.22 from CUR 228."""
+
+import pytest
+
+from blueprints.codes.cur.cur_228.formula_2_22 import Form2Dot22MenardStiffness
+
+
+class TestForm2Dot22MenardStiffness:
```

Comment:
I would probably add some tests for the raise if negative checks. To make sure it's captured